src/nft/components/bag/profile/ListingButton.tsx

```diff
@@ -103,8 +103,9 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
       return asset.expirationTime != null && isNaN(asset.expirationTime)
     })
     const overMaxExpiration = sellAssets.some((asset) => {
-      return asset.expirationTime != null && asset.expirationTime - Date.now() > ms`180 days`
+      return asset.expirationTime != null && asset.expirationTime * 1000 - Date.now() > ms`180 days`
```

Comment:
could we use the `ms` module to handle `asset.expirationTime`?

Reply:
All the marketplaces require the expiration time given in seconds as opposed to ms which is which i mirrored in expirationTime